lang/python/python-pyxdg/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pyxdg
+PKG_VERSION:=0.26
+PKG_RELEASE:=1
+PKG_LICENSE:=LGPL-2.0
+
+PKG_SOURCE:=pyxdg-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pyxdg
+PKG_HASH:=fe2928d3f532ed32b39c32a482b54136fe766d19936afc96c8f00645f9da1a06
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-pyxdg-$(PKG_VERSION)
+
+PKG_BUILD_DEPENDS:=python
```

Comment:
I believe because `python` is already a runtime dependency, this `PKG_BUILD_DEPENDS` isn't necessary.